tests/src/rules/extensions.js

```diff
@@ -367,6 +368,11 @@ ruleTester.run('extensions', rule, {
           line: 4,
           column: 31,
         },
+        {
+          message: 'Missing file extension for "@/scoped/bav/import"',
+          line: 7,
+          column: 25,
+        },
```

Comment:
I'd expect `ignorePackages` to suppress this warning, since this is a package.

Reply:
I've edited the case, thanks.